packages/jest-runner/src/index.ts

```diff
@@ -90,7 +91,7 @@ class TestRunner {
                 eventName: string,
                 args: Array<unknown>,
               ) => {
-                this.eventEmitter.emit(eventName, args);
+                this.eventEmitter.emit(eventName, deepCyclicCopy(args));
```

Comment:
All that work and it's a one line fix? 😅

Reply:
Yeah lol

Reply:
Not kidding, finding that one-liner was painful. Very